src/markdown-pages/automate-workflows/kubernetes-helm-deployment.mdx

```diff
@@ -0,0 +1,150 @@
+
+Replace `YOUR_NEW_RELIC_LICENSE_KEY` and `CLUSTER_NAME` before running the following command:
+```bash lineNumbers=false
+helm install newrelic-bundle newrelic/nri-bundle \
+ --set global.licenseKey=YOUR_NEW_RELIC_LICENSE_KEY \
+ --set global.cluster=CLUSTER_NAME \
+ --namespace=default \
+ --set newrelic-infrastructure.privileged=true \
+ --set ksm.enabled=true \
+ --set prometheus.enabled=true \
+ --set kubeEvents.enabled=true \
+ --set logging.enabled=true
+```
+You should see something similar to the following:
+```no-highlighting lineNumbers=false
+NAME: newrelic-bundle
+LAST DEPLOYED: Wed Aug 19 09:04:52 2020
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+
+Check if the New Relic agents have been deployed:
+```no-highlighting lineNumbers=false
+kubectl get daemonsets,pods
+```
+
+This should look similar to:
+```no-highlighting lineNumbers=false
+NAME                                                     DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+daemonset.apps/newrelic-bundle-newrelic-infrastructure   1         1         1       1            1           <none>          2m53s
+daemonset.apps/newrelic-bundle-newrelic-logging          1         1         1       1            1           <none>          2m53s
+NAME                                                          READY   STATUS      RESTARTS   AGE
+pod/newrelic-bundle-kube-state-metrics-69ff8cfb74-rgjc5       1/1     Running     0          2m53s
+pod/newrelic-bundle-newrelic-infrastructure-z8ddb             1/1     Running     0          2m53s
+pod/newrelic-bundle-newrelic-logging-wp22p                    1/1     Running     0          2m53s
+pod/newrelic-bundle-nri-kube-events-f9d5bb944-kcxxf           2/2     Running     0          2m53s
+pod/newrelic-bundle-nri-metadata-injection-66d76c868b-xrcq8   1/1     Running     0          2m53s
+pod/newrelic-bundle-nri-metadata-injection-job-rszw5          0/1     Completed   0          2m53s
+pod/newrelic-bundle-nri-prometheus-569689b7cb-pnddg           1/1     Running     0          2m53s
+```
+</Step>
+<Step>
+
+## Use the New Relic Kubernetes cluster explorer
+
+Open the [Kubernetes cluster explorer in New Relic One](https://one.newrelic.com/launcher/k8s-cluster-explorer-nerdlet.cluster-explorer-launcher?pane=eyJuZXJkbGV0SWQiOiJrOHMtY2x1c3Rlci1leHBsb3Jlci1uZXJkbGV0Lms4cy1jbHVzdGVyLWxpc3QifQ==)
+
+You should see something like:
+
+![Launcher artifact with icon.png](../../images/kubernetes-helm-deployment/k8s-cluster-explorer.png)
+
+Want to know more? [Learn how to navigate the Kubernetes cluster explorer](https://docs.newrelic.com/docs/integrations/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer)
+
+</Step>
+</Steps>
+
+## Try it out now
+
+We have a Kubernetes test cluster ready for you in 2 minutes. By following this on-line tutorial, you will learn how to deploy the New Relic Helm charts.
+
+**Some tips to use the on-line tutorial window:**
```

Comment:
I'd suggest making this a  H2 header

Reply:
Good idea, I used an H3, so it fits under the H2 of the 'try it out now' headerr